### ISSUE TITLE
Reduce embedding dimension

### ISSUE THREAD

**Issue by aigagror** [aigagror/MuZeroGoJax#82]
Maybe try 8.

So getting all transition value logits won't be so expensive
